Cannot use scope package for theme.

<!-- Issuehunt Badges -->
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%2460%20Funded-%2300A156.svg" />](https://issuehunt.io/r/saberland/saber/issues/360)
<!-- /Issuehunt Badges -->


<!-- Please don't delete this template or we'll close your issue -->
<!-- Before creating an issue please make sure you are using the latest version of Saber. -->

<!-- Please confirm you will submit an issue. -->
<!-- Issues which contain questions or support requests will be closed. -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

- [x] I confirm that this is an issue rather than a question.

<!-- Please ask questions via following several ways. -->
<!-- https://chat.saber.land/ -->
<!-- https://stackoverflow.com/questions/ask?tags=saberjs -->

## Bug report

You can't use a scoped package as a theme. If you try to set one, you get the following error:

```info Using config file: docs/saber-config.js
error Error: Cannot find module 'saber-theme-@skewedaspect/saber-theme-minima/package.json'
Require stack:
- /Users/ccase/devel/armtech/saber-test/docs/noop.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at resolveFileName (/Users/ccase/devel/armtech/saber-test/node_modules/saber/node_modules/resolve-from/index.js:29:39)
    at resolveFrom (/Users/ccase/devel/armtech/saber-test/node_modules/saber/node_modules/resolve-from/index.js:43:9)
    at module.exports (/Users/ccase/devel/armtech/saber-test/node_modules/saber/node_modules/resolve-from/index.js:46:47)
    at module.exports (/Users/ccase/devel/armtech/saber-test/node_modules/saber/lib/utils/resolvePackage.js:19:5)
    at Saber.prepare (/Users/ccase/devel/armtech/saber-test/node_modules/saber/lib/index.js:135:20)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
    at async Saber.serve (/Users/ccase/devel/armtech/saber-test/node_modules/saber/lib/index.js:349:5)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Process finished with exit code 1
```

#### Version

v0.7.0

#### Steps to reproduce

Easiest way to reproduce:
* https://github.com/Morgul/saber-scope-bug

Basically, just try to use a theme published under a scoped package.

#### What is expected?

The theme just works.

#### What is actually happening?

It does not work.

#### Other relevant information

- Result of running `saber -v`:
  - `saber/0.7.0 darwin-x64 node-v12.4.0`
- Is Saber a global or local install?
  - local
- Which package manager did you use for the install?
  - yarn
- Does this issue occur when all plugins are disabled?
  - yes


<!-- Issuehunt content -->

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>


### Backers (Total: $60.00)

- [<img src='https://avatars3.githubusercontent.com/u/44827199?v=4' alt='issuehunt' width=24 height=24> issuehunt](https://issuehunt.io/u/issuehunt) ($60.00)
### Submitted pull Requests
- [#361 fix: Scoped packages can't be used as themes](https://issuehunt.io/r/saberland/saber/pull/361)
---

#### [Become a backer now!](https://issuehunt.io/r/saberland/saber/issues/360)
#### [Or submit a pull request to get the deposits!](https://issuehunt.io/r/saberland/saber/issues/360)
### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/saberland/saber/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>
<!-- /Issuehunt content-->

--- a/packages/saber/lib/utils/__test__/resolvePackage.test.js
+++ b/packages/saber/lib/utils/__test__/resolvePackage.test.js
@@ -1,0 +1,18 @@
+const resolvePackage = require('../resolvePackage')
+
+test('resolvePackage', () => {
+  expect(resolvePackage('foo', { cwd: false })).toBe('foo')
+  expect(resolvePackage('@foo/bar', { cwd: false })).toBe('@foo/bar')
+  expect(resolvePackage('foo', { prefix: 'prefix-', cwd: false })).toBe(
+    'prefix-foo'
+  )
+  expect(resolvePackage('prefix-foo', { prefix: 'prefix-', cwd: false })).toBe(
+    'prefix-foo'
+  )
+  expect(resolvePackage('@foo/bar', { prefix: 'prefix-', cwd: false })).toBe(
+    '@foo/prefix-bar'
+  )
+  expect(
+    resolvePackage('@foo/prefix-bar', { prefix: 'prefix-', cwd: false })
+  ).toBe('@foo/prefix-bar')
+})

--- a/packages/saber/lib/utils/resolvePackage.js
+++ b/packages/saber/lib/utils/resolvePackage.js
@@ -5,22 +5,35 @@ const resolveFrom = require('resolve-from')
 const LOCAL_PATH_RE = /^[./]|(^[a-zA-Z]:)/
 
 /**
+ *
+ * @param {string} input
+ * @param {string=} prefix
+ */
+const addPrefix = (input, prefix) => {
+  if (!prefix) return input
+  if (input.startsWith('@')) {
+    return input.replace(new RegExp(`^@(\\w+)/(${prefix})?`), `@$1/${prefix}`)
+  }
+
+  return input.startsWith(prefix) ? input : `${prefix}${input}`
+}
+
+/**
  * @param {string} input
  * @param {Object} options
- * @param {string} [options.cwd=process.cwd()]
+ * @param {string|false} [options.cwd=process.cwd()]
  * @param {string=} options.prefix
  */
 module.exports = (input, { cwd = process.cwd(), prefix } = {}) => {
   if (LOCAL_PATH_RE.test(input)) {
-    return path.resolve(cwd, input)
+    return cwd === false ? input : path.resolve(cwd, input)
   }
 
-  return path.dirname(
-    resolveFrom(
-      cwd,
-      `${
-        !prefix || input.startsWith(prefix) ? input : `${prefix}${input}`
-      }/package.json`
-    )
-  )
+  input = addPrefix(input, prefix)
+
+  if (cwd === false) {
+    return input
+  }
+
+  return path.dirname(resolveFrom(cwd, `${input}/package.json`))
 }

--- a/packages/saber/lib/utils/resolvePackage.js
+++ b/packages/saber/lib/utils/resolvePackage.js
@@ -5,12 +5,13 @@ const resolveFrom = require('resolve-from')
 const LOCAL_PATH_RE = /^[./]|(^[a-zA-Z]:)/
 
 /**
- *
+ * Add prefix to package name
  * @param {string} input
  * @param {string=} prefix
  */
 const addPrefix = (input, prefix) => {
   if (!prefix) return input
+
   if (input.startsWith('@')) {
     return input.replace(new RegExp(`^@(\\w+)/(${prefix})?`), `@$1/${prefix}`)
   }

